AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -4304,6 +4304,14 @@ worldometers.info#?#div:matches-css(border:3px):has(> div > div > div > iframe[s
 ||womenshealthmag.com/*/files/wh-newsletter-signup$image
 ||x.mail.bonnier-subscriptions.com^$domain=flyingmag.com|popsci.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/129719
+substack.com#$#body { overflow: visible!important; }
+substack.com##.subscribe-dialog-scroll-modal-scroll-capture
+substack.com##.subscribe-dialog-scroll-modal
+substack.com##.subscribe-dialog-above-the-fold
+substack.com##.subscribe-dialog-friendly-neighborhood-scroll-capture
+substack.com##.subscribe-dialog-friendly-neighborhood
```

Comment:
Changed existing rule `substack.com##div[class^="subscribe-"]`

Reply:
About popups - need more "substacks". Not a good idea to add unique rules for thousands of blogs.

Reply:
Because we don't need unnecessary rules. We need at least 5 blogs with the same popup - `.default-intro-popup`. You removed useful rule. Also both rules must be CSS to prevend breakage on plathowm without CSS support.